server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/get/GetSnapshotsRequest.java

```diff
@@ -130,6 +138,9 @@ public void writeTo(StreamOutput out) throws IOException {
             out.writeEnum(sort);
             out.writeVInt(size);
             order.writeTo(out);
+            if (out.getVersion().onOrAfter(NUMERIC_PAGINATION_VERSION)) {
```

Comment:
Would we not want to fail if offset is non-zero when talking to a too old version? Just like 3 lines down.

Reply:
++ added handling for that thanks!